appcues/src/main/java/com/appcues/debugger/AppcuesDebuggerManager.kt

```diff
@@ -26,6 +26,7 @@ import kotlinx.coroutines.launch
 import org.koin.core.scope.Scope
 import kotlin.coroutines.CoroutineContext
 
+@Suppress("TooManyFunctions")
```

Comment:
I had to have a way to pass the `reset()` through here, now that no event is triggered. That tipped this class over the 15 function limit. Since its a debugger class, I feel not critical, but don't love suppressing, and don't have a better idea at the moment.

Reply:
not sure if Applies, there is a DefaultLifetracker that we dont necessarily need to override all methods, maybe could be used here to reduce unused lifecycle methods?

Reply:
It looks like that is for listening to the lifecycle events of a single owner (i.e. Activity) through the Activity's `lifecycle`
```kotlin
lifecycle.addObserver(x)
```

But in this case, we are actually trying to listen to all activities in the application through 
```kotlin
application.registerActivityLifecycleCallbacks(this)
```

So I don't think it can be used in this case.